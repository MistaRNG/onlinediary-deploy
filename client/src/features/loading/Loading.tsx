import { Spinner } from "@blueprintjs/core";

const Loading: React.FC = () => {
  return <Spinner />;
};

export default Loading;
