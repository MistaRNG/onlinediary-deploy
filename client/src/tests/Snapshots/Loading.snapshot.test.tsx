import React from 'react';
import { render } from '@testing-library/react';
import Loading from '../../features/loading/Loading';
import { describe, it, expect } from '@jest/globals';

describe('Loading Component', () => {
  it('should match snapshot', () => {
    // ACT
    const { asFragment } = render(<Loading />);

    // ASSERT
    expect(asFragment()).toMatchSnapshot();
  });
});
