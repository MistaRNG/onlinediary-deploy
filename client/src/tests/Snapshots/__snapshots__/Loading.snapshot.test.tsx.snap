// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should match snapshot 1`] = `
<DocumentFragment>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    class="bp4-spinner"
    role="progressbar"
  >
    <div
      class="bp4-spinner-animation"
    >
      <svg
        height="50"
        stroke-width="8.00"
        viewBox="1.00 1.00 98.00 98.00"
        width="50"
      >
        <path
          class="bp4-spinner-track"
          d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90"
        />
        <path
          class="bp4-spinner-head"
          d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90"
          pathLength="280"
          stroke-dasharray="280 280"
          stroke-dashoffset="210"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
